unit-tests/py/common.py

```diff
@@ -0,0 +1,117 @@
+import os, sys, subprocess, traceback
+import pyrealsense2 as rs
+
+n_assertions = 0
+n_failed_assertions = 0
+n_tests = 0
+n_failed_tests = 0
+test_failed = False
+
+# If this is the first time running this script we set the wanted enviroment, However it is impossible to change the current running 
+# environment so we rerun the script in a child process that inherits the enviroment we set
+def set_env_vars(env_vars):
+    if len(sys.argv) < 2:
+        for env_var, val in env_vars.items():
+            os.environ[env_var] = val
+        sys.argv.append("rerun")
+        cmd = ["py", "-3"] + sys.argv
+        p = subprocess.run( cmd, stderr=subprocess.PIPE, universal_newlines=True )
+        exit(p.returncode)
+
+# Returns the first device that was found, if no device is found the test is aborted
+def get_first_device():
+    c = rs.context()
+    if not c.devices.size():  # if no device is connected we don't run the AC test
+        print("No device found, aborting AC test")
+        exit(0)
+    return c.devices[0]
+
+# Functions for asserting test cases:
+# Receives an expression which is an assertion. If false the assertion failed
+def require(exp):  
+    global n_assertions, n_failed_assertions, test_failed
+    n_assertions += 1
+    if not exp:
+        n_failed_assertions += 1
+        test_failed = True
+        print("Require failed")
```

Comment:
Don't you want to show the value of `exp`?
I suggest you use a common function for showing errors, like `error` in my script -- it would allow easy formatting changes, coloring if possible, etc.

Reply:
require is used for Boolean expressions so if it failed the value is always False.
I could try to create a common function but each type of require prints different things so I'm not sure it will help

Reply:
No need for other functions -- but the value passed in won't always be False. It can be None, 0, or even another class that defines `__nonzero__`...

Reply:
ok, I'll add to the printed message